app/src/main/java/com/door43/translationstudio/core/TargetTranslationMigrator.java

```diff
@@ -351,48 +355,93 @@ private static boolean v2( File path) throws Exception {
         return true;
     }
 
-   /**
-    * Merges chunks found in the target translation projects that do not exist in the source translation
-    * to a sibling chunk so that no data is lost.
-    * @param library
-    * @param translationSlugs target translations to merge
-    * @return
-    */
-    public static boolean migrateChunkChanges(final Translator translator, final Library library, final String[] translationSlugs) {
-        boolean mergeSuccess = true;
-        for (String translationSlug : translationSlugs) {
-            final TargetTranslation targetTranslation = translator.getTargetTranslation(translationSlug);
-            boolean success = migrateChunkChanges(library, targetTranslation);
-            mergeSuccess = mergeSuccess && success;
+    /**
+     * convert project from USX to USFM
+     * @param manifest
+     * @param targetTranslationDir
+     * @return
+     */
+    private static boolean migrateFromUSXtoUSFM(final JSONObject manifest, final File targetTranslationDir) {
+
+        Translator translator = AppContext.getTranslator();
+        TargetTranslation targetTranslation = TargetTranslation.open(targetTranslationDir);
+        if(null != targetTranslation) {
+
+            TranslationFormat format = readTranslationFormat(targetTranslation);
+            if(TranslationFormat.USFM == format) {
```

Comment:
We should not depend on global variables, but instead hardcode the current value of the global value here. Otherwise if we change it in the future this will break